README.md

```diff
@@ -285,6 +287,33 @@ fastify.listen(3000, err => {
 })
 ```
 
+#### Example black-listing tokens
```

Comment:
```suggestion
#### Example trusted tokens
```

Reply:
@francisbrito could check this?

Reply:
Yes, sorry. I switched computer in the last week and I thought I already had uploaded this changes before switching